tests/unit/AddStaffNotationDialog.spec.ts

```diff
@@ -227,10 +237,10 @@ describe('AddStaffNotationDialog', () => {
     await Vue.nextTick()
 
     // Should not validate if plan of arrangement is not checked
-    expect(wrapper.find('#court-order').text()).not.toContain('A Court Order number is required')
+    expect(wrapper.find('#court-order-poa').text()).not.toContain('A Court Order number is required')
```

Comment:
This targets the local/shared CourtOrderPoa component without requiring a separate id for it.